images/kubeadm/runner

```diff
@@ -21,14 +21,6 @@ if [ ! -e test-infra ]; then
   git clone https://github.com/kubernetes/test-infra
 fi
 
-if [ ! -e kubernetes-anywhere ]; then
```

Comment:
here we the `-e` check in the runner. i guess using `--repo` for the jobs instead would pull every time?
probably OK for smaller repos.

Reply:
this doesn't really make sense, I don't think this check would be true under any circumstance